docs/jruby-limitations.md

```diff
@@ -1,6 +1,6 @@
 # Cucumber and JRuby limitations
 
-`cucumber` can be executed on `JRuby` (tested with `9.2.8.0`), although some of the features
+`cucumber` can be executed on `JRuby` (tested with `9.1`and `9.2`), although some of the features
```

Comment:
```suggestion
`cucumber` can be executed on `JRuby` (tested with `9.1` and `9.2`), although some of the features
```

Reply:
well, it seems we have even more problems with JRuby 9.1 (apparently [it's failing with language `he`](https://app.circleci.com/jobs/github/cucumber/cucumber-ruby/4384)).
I guess it's safer to just indicate `9.2`